Pollo/ViewControllers/WebViewController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  WebViewController.swift
+//  Pollo
+//
+//  Created by Gonzalo Gonzalez on 9/23/20.
+//  Copyright © 2020 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+import FutureNova
+import WebKit
+
+class WebViewController: UIViewController {
+
+    var ssoWebView: WKWebView!
+    let sessionTokenHandlerName: String = "sessionTokenHandler"
+    let ssoEndpoint: String = "http://pollo-dev.cornellappdev.com/api/v2/auth/saml/cornell/"
```

Comment:
is this the permanent ssoEndpoint (i'm guessing no, since it's `pollo-dev`)? 

regardless, can we not put the plain API endpoint in the code, and instead use the Keys.swift / Keys.plist?